lib/Compiler.js

```diff
@@ -59,13 +64,29 @@ function compileSolc(logger, contractFiles, cb) {
           compiled_object[className].functionHashes = contract.hashes;
           compiled_object[className].abiDefinition = JSON.parse(contract.abi);
           compiled_object[className].filename = fileName;
+
+          
         }
 
         fileCb();
       });
     },
     function (err) {
-      cb(err, compiled_object);
+      cb(err, compiled_object);  
+      
+      embark.events.on("outputDone", function() {
+        if(outputBinary){
+          Object.keys(compiled_object).map(function(className, index) {
+            fs.writeFile(outputDir + className + ".bin", compiled_object[className].bin, (err) => {
```

Comment:
Are slashes already present inside outputDir and className?

Reply:
This should use the [`path.join`](https://nodejs.org/api/path.html#path_path_join_paths) API to make it OS agnostic.